app/assets/javascripts/controllers/container_dashboard/util/dashboard-utils-factory.js

```diff
@@ -97,8 +97,28 @@ angular.module('miq.util').factory('dashboardUtilsFactory', function() {
   var parseDate = function(date) {
     myDate = Date.parse(date);
     return isNaN(myDate) ? date : myDate
-  }
+  };
+  var iconClassForProvider = function(provider) {
+    switch (provider) {
+      case "openshift":
+        iconClass = "pficon pficon-openshift";
```

Comment:
you can combine  lines 103 to lines 108. just put the 2 case statements together one after another